Vagrantfile

```diff
@@ -165,26 +151,21 @@ Vagrant.configure(2) do |config|
     ${HOME}/bin/direnv allow ${HOME}/scf
   SHELL
 
-  config.vm.provision "shell", privileged: false, inline: <<-SHELL
-    set -e
-
+  # Install common and dev tools
+  config.vm.provision :shell, privileged: true, inline: <<-SHELL
+    set -o errexit -o xtrace -v
     # Get proxy configuration here
-    source /etc/environment
-    export no_proxy http_proxy https_proxy NO_PROXY HTTP_PROXY HTTPS_PROXY
-
-    # Install development tools
-    (
-      cd "${HOME}/scf"
-      ${HOME}/bin/direnv exec ${HOME}/scf/bin/dev/install_tools.sh
-    )
-
+    export HOME=/home/vagrant
+    export PATH=$PATH:/home/vagrant/bin
+    cd "${HOME}/scf"
+    bash ${HOME}/scf/bin/common/install_tools.sh
```

Comment:
This does need to be privileged, because in the vagrant box, the tools are going into `/usr/local/bin`. This is useful because if we want to use any of these tools from a non-interactive, non-login shell, they need to be in a directory which will be in PATH.